### DISCUSSION TITLE
Downloading replays but without actual play (score only)

### DISCUSSION THREAD

**Discussion by changingnicknamebrb** [ppy/osu#24408] [Ideas]
I tried to search this discussion but didn't find anything about it.

I wanted to download my old plays from years ago for nostalgic value. I know that they're saved in score-only state, because i wasn't a good player back then, so I didn't expect to download a full replay. osu!stable lets you download a score without a replay, but I didn't find this feature in osu!lazer. Should I wait for this feature to be implemented or just use osu!stable to download scores and import them to osu!lazer?

**Comment by peppy** [comment]
I wasn't aware that stable can do this. Is it just F2 from results screen? And you want your online best score to appear in local ranking?

**Reply by changingnicknamebrb** [reply]
Yes and yes.

Also a bit later I tried to export this type of replay into lazer, it exported replay normally but gave me ability to "watch" a replay. When I tried to do that, it was just a static cursor with no circle hits. It was done in offline, maybe it did something about this situation.

**Comment by blakeplusplus** [comment]
As someone who is playing again after not playing for 10 years (jeez time flies), it'd be really great to be able to compete against my old scores on the local leaderboard. Hope this gets added.
